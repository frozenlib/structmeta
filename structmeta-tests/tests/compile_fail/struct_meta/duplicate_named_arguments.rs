#[structmeta_tests::attr_required_name_value(value = 10, value = 20)]
fn my_func() {}

fn main() {}
