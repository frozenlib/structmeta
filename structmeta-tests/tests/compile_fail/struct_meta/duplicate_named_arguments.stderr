error: parameter `value` speficied more than once
 --> $DIR/duplicate_named_arguments.rs:1:58
  |
1 | #[structmeta_tests::attr_required_name_value(value = 10, value = 20)]
  |                                                          ^^^^^
